docs/introduction_tutorial/intro_tutorial_part2.ipynb

```diff
@@ -0,0 +1,1228 @@
+{
```

Comment:
<p>let's use the term "Pangeo Forge Cloud" and avoid the word "deployment", which in my experience doesn't mean much to users.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pangeo-forge/pangeo-forge-recipes/pull/293/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>